paddle/fluid/operators/fake_quantize_op.cc

```diff
@@ -217,12 +219,19 @@ class FakeQuantizeAbsMaxOpMaker : public framework::OpProtoAndCheckerMaker {
                                 bit_length));
         });
     AddComment(R"DOC(
-FakeQuantize operator
+This is a Base Op which support FakeQuantAbsMaxOpMaker and FakeQuantDequantAbsMaxOpMaker.
+FakeQuantAbsMaxOp operator is used in the dynamic quantization.
 
 $$scale = max(abs(X))$$
 $$range = 2^{bit_length - 1} - 1$$
 $$Out = round(X/scale * range)$$
 
+FakeQuantDequantAbsMaxOp operator do the abs_max quant and then dequant.
```

Comment:
Tiny issue:

"FakeQuantDequantAbsMaxOp operator does"

do -> does

Same as following

Reply:
Thx. Can I fix the syntax error in the next pr?

Reply:
Agree